hive-bigquery-connector-common/src/main/java/com/google/cloud/hive/bigquery/connector/config/HiveBigQueryConfig.java

```diff
@@ -196,21 +204,15 @@ public static Map<String, String> hadoopConfigAsMap(Configuration conf) {
     return configMap;
   }
 
-  private static Optional<String> getAnyOption(
-      String key, Configuration conf, Map<String, String> tableParameters) {
-    // TO-DO: here we choose conf value over table value, any issue?
-    String value = conf.get(key);
-    if (value == null && tableParameters != null) {
-      value = tableParameters.get(key);
-    }
-    return Optional.fromNullable(value);
+  private static Optional<String> getOption(String key, Configuration conf) {
+    return Optional.fromNullable(conf.get(key));
   }
 
   private static Optional<String> getAnyOption(
```

Comment:
the getAnyOption sounds like returns Optional<Any>, can we name it getOption? overload the getOption